avm/res/api-management/service/main.bicep

```
@@ -54,7 +63,7 @@ param restore bool = false
 @description('Optional. Array of role assignments to create.')
 param roleAssignments roleAssignmentType
 
-@description('Optional. The pricing tier of this API Management service.')
+@description('Optional. The pricing tier of this API Management service. Default is Premium.')
```

This is representative of all parameters - the default should not be part of the name because
- in the readme.md it's automatically added from the default
- in Bicep linter (when using the module) it's automatically suggested
- It adds another location (per parameter) where you'd need to maintain the value - so it could run stale

```suggestion
@description('Optional. The pricing tier of this API Management service.')
```

---

This would be the same comment across all modules, btw :) I'll refrain from commenting every instance though